Networking/NetworkingTests/Remote/ReceiptRemoteTests.swift

```diff
@@ -85,4 +85,31 @@ final class ReceiptRemoteTests: XCTestCase {
         let received = try XCTUnwrap(request.parameters["force_new"] as? String)
         assertEqual("false", received)
     }
+
+    func test_retrieveReceipt_when_expiration_date_is_unset_then_returns_default_2_days() throws {
```

Comment:
Since we are testing for the parameter value, not the return/completion value, what do you think about renaming the method for clarity? 
```suggestion
    func test_retrieveReceipt_when_expiration_date_is_unset_then_sends_expiration_days_parameter_as_default_2_days() throws {
```
The same applies for the other unit test as well.

Reply:
Sounds good, updated on https://github.com/woocommerce/woocommerce-ios/pull/11957/commits/11f10b88416f64491a21d196d154a65155b987a3